syncstorage-postgres/src/pool.rs

```diff
@@ -58,24 +58,24 @@ fn run_embedded_migrations(database_url: &str) -> DbResult<()> {
 
     conn2.run_pending_migrations(MIGRATIONS)?;
     Ok(())
-    */
```

Comment:
Multiple tests are simultaneously trying to run the migrations, causing the tests to fail.

Reply:
I wouldn't expect most if any tests to work yet until more of the impl is stubbed out. What did you try running? May be that the mutability issue experienced in MySQL is not the case here and I can remove the `conn2 = conn` here. Let me push that as a change and see if it's different for you.

Reply:
To be clear, I'm not suggesting the tests are failing due to any impl.  I'm saying the tests _error out_ on the migrations step, which is enabled in this patch.

Reply:
I learned from @pjenvey that we use `RUST_TEST_THREADS=1` for our unit tests (see the `test` and `test_with_coverage` make targets).  That fixes this error.